collect_app/src/main/java/org/odk/collect/android/activities/GoogleSheetsUploaderActivity.java

```diff
@@ -302,7 +306,7 @@ public void onRequestPermissionsResult(int requestCode,
      */
     @Override
     public void onPermissionsGranted(int requestCode, List<String> list) {
-        // Do nothing.
+//        getResultsFromApi();
```

Comment:
Were these things you were trying? Better not to leave comments like this unless there's an explanation with them!